MovieQuiz/Services/QuestionFactory.swift

```diff
@@ -9,67 +9,105 @@ import Foundation
-            correctAnswer: true),
-        QuizQuestion(
-            image: "Deadpool",
-            text: "Рейтинг этого фильма больше чем 6?",
-            correctAnswer: true),
-        QuizQuestion(
-            image: "The Green Knight",
-            text: "Рейтинг этого фильма больше чем 6?",
-            correctAnswer: true),
-        QuizQuestion(
-            image: "Old",
-            text: "Рейтинг этого фильма больше чем 6?",
-            correctAnswer: false),
-        QuizQuestion(
-            image: "The Ice Age Adventures of Buck Wild",
-            text: "Рейтинг этого фильма больше чем 6?",
-            correctAnswer: false),
-        QuizQuestion(
-            image: "Tesla",
-            text: "Рейтинг этого фильма больше чем 6?",
-            correctAnswer: false),
-        QuizQuestion(
-            image: "Vivarium",
-            text: "Рейтинг этого фильма больше чем 6?",
-            correctAnswer: false)
-    ]
+    //    private let questions = [
+    //        QuizQuestion(
+    //            image: "The Godfather",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: true),
+    //        QuizQuestion(
+    //            image: "The Dark Knight",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: true),
+    //        QuizQuestion(
+    //            image: "Kill Bill",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: true),
+    //        QuizQuestion(
+    //            image: "The Avengers",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: true),
+    //        QuizQuestion(
+    //            image: "Deadpool",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: true),
+    //        QuizQuestion(
+    //            image: "The Green Knight",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: true),
+    //        QuizQuestion(
+    //            image: "Old",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: false),
+    //        QuizQuestion(
+    //            image: "The Ice Age Adventures of Buck Wild",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: false),
+    //        QuizQuestion(
+    //            image: "Tesla",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: false),
+    //        QuizQuestion(
+    //            image: "Vivarium",
+    //            text: "Рейтинг этого фильма больше чем 6?",
+    //            correctAnswer: false)
+    //    ]
     
-    init(delegate: QuestionFactoryDelegate? = nil) {
+    // MARK: - Private properties
+    private let moviesLoader: MoviesLoading
+    private var mostPopularMovie = [MostPopularMovie]()
```

Comment:
Эта форма предпочтительнее
```suggestion
    private var mostPopularMovie: [MostPopularMovie] = []
```